debezium-server-bigquery-sinks/src/main/java/io/debezium/server/bigquery/DebeziumBigqueryEvent.java

```diff
@@ -102,9 +104,20 @@ private static ArrayList<Field> getBigQuerySchemaFields(JsonNode schemaNode, Boo
           fields.add(Field.of(fieldName, StandardSQLTypeName.BOOL));
           break;
         case "string":
-          fields.add((castDeletedField && Objects.equals(fieldName, "__deleted"))
-              ? Field.of(fieldName, StandardSQLTypeName.BOOL)
-              : Field.of(fieldName, StandardSQLTypeName.STRING));
+          switch (fieldSemanticType) {
+            case "io.debezium.data.Json":
+              // NOTE! Not supported by batch load, BQ batch write api has issue with loading escaped json field name!
+              fields.add(Field.of(fieldName, StandardSQLTypeName.STRING));
+              break;
+            case "io.debezium.time.ZonedTimestamp":
+              fields.add(Field.of(fieldName, StandardSQLTypeName.TIMESTAMP));
```

Comment:
Won't we lose a zone information here?

Reply:
Added a test on it. Time zone information is kept correct. Debezium is always extracting it as UTC and BQ importing as UTC. Do you see any issues? Or a way to test it? 
Please feel free to send me or request